.circleci/config.yml

```diff
@@ -15,34 +15,44 @@ parameters:
     type: string
     default: "dev:alpha"
 
-commands:
```

Comment:
Could this command still be useful for v1 tests? It could be potentially renamed to `check-latest-v1-version-installed` for clarity?

Reply:
Addded directly within the v1 test job.
https://app.circleci.com/pipelines/github/CircleCI-Public/aws-cli-orb/153/workflows/1c03accd-43d1-4752-aacd-8a6b928b7b8e/jobs/392